google_compute_engine_oslogin/authorized_keys/authorized_keys.cc

```diff
@@ -45,7 +45,7 @@ int main(int argc, char* argv[]) {
     // Return 0 if the user is not an oslogin user. If we returned a failure
     // code, we would populate auth.log with useless error messages.
     return 0;
-  } else if (user_response.empty() || http_code == 500) {
+  } else if (user_response.empty() || http_code != 200) {
     return 1;
```

Comment:
nit: Like what you did in other modules, this probably can be merged with the above `if (!HttpGet())` check